src/Reader/Entry/AbstractEntry.php

```diff
@@ -173,14 +175,15 @@ public function getExtensions()
      * Return an Extension object with the matching name (postfixed with _Entry)
      *
      * @param  string $name
-     * @return Reader\Extension\AbstractEntry
+     * @return null|Reader\Extension\AbstractEntry
```

Comment:
This change is a bit suspicious :thinking:

Reply:
Same comment as above - the code was already doing this, but wasn't documented as doing it.